app/models/conversion_host.rb

```diff
@@ -75,12 +77,13 @@ def check_conversion_host_role
     tag_resource_as('disabled')
   end
 
-  def enable_conversion_host_role
+  def enable_conversion_host_role(v2v_vddk_package_url = nil)
     install_conversion_host_module
+    v2v_vddk_package_url |= "http://#{resource.ext_management_system.hostname}/vddk/VMware-vix-disklib-stable.tar.gz"
```

Comment:
I'm guessing you want `||=` here.

Reply:
yep, thanks
should have added specs first